js/src/lib/interfaces/Libraries.ts

```diff
@@ -294,6 +295,18 @@ const fastai = (model: ModelData) =>
 
 learn = from_pretrained_fastai("${model.id}")`;
 
+const mindspore = (model: ModelData) => {
+	const architecture = model.config.architectures[0].toLowerCase();
+	if (model.tags?.includes("image-classification")){
```

Comment:
In case it's not an image classification model, we should have some fallback/unknown snippet (e.g. https://github.com/hellowaywewe/hub-docs/blob/main/js/src/lib/interfaces/Libraries.ts#L82). Just to confirm, the given snippet does not work unless it's a `image-classification` model?

Reply:
@osanseviero Yes, you are right, currently we provide image-classification only as a test, but more tasks will be supported later. I have submitted a new PR, please review it. (https://github.com/huggingface/hub-docs/pull/673)

Reply:
Sorry, this PR (https://github.com/huggingface/hub-docs/pull/677)